File: src/classes/api/req_res/add-timeis-response.ts
'use strict'

import { TimeIs } from '@/classes/datas/time-is'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddTimeisResponse extends GkillAPIResponse {

    added_timeis: TimeIs

    added_kyou: Kyou | null

    constructor() {
        super()
        this.added_timeis = new TimeIs()
        this.added_kyou = null
    }

}

File: src/server/gkill/api/req_res/add_time_is_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type AddTimeIsResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	AddedTimeis *reps.TimeIs `json:"added_timeis"`

	AddedKyou *reps.Kyou `json:"added_kyou"`
}
